cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -37,6 +40,10 @@ function Autocomplete( element, opts ) {
   let _suggestions;
   let _isVisible;
   let _selection;
+  let _maxLengthExceeded = false;
+
+  /* TODO: Is there a better way to get this to use later to dispatch the event? */
```

Comment:
Have you confirmed changing `_this.dispatchEvent` to `this.dispatchEvent` on line 165 doesn't work? It probably will be referencing `_input` huh.

Reply:
Yeah, `this` in that context is the `_input` HTML node. Given that, is there a more preferred way to do this? Looks like we do `const _this = this` in a couple of places (like [cfgov/unprocessed/js/modules/TabTrigger.js#L31](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/js/modules/TabTrigger.js#L31), but I'm happy to change it to whatever if you have a preference, @anselmbradford.

Reply:
That seems fine. You can also do `_instance` if that's clearer https://github.com/cfpb/design-system/blob/main/packages/cfpb-forms/src/organisms/Multiselect.js#L66 

Alternative, you can use bind to set the value of `this` by created a binded alias of the `_toggleMaxLengthError` function, like [this](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/js/modules/behavior/FlyoutMenu.js#L55).

Reply:
I went with `_instance` 1e506c1—that felt like the most readable option.